db/schema.rb

```diff
@@ -16,7 +16,7 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "bookings", force: true do |t|
+  create_table "bookings", force: :cascade do |t|
```

Comment:
"`SchemaDumper` uses `force: :cascade` on `create_table`. This makes it possible to reload a schema when foreign keys are in place."

From Rails 4.2 release notes.